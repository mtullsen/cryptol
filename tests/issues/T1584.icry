take`{5} [ x+y | x <- [(0:Integer)...], y <- [1] ]
