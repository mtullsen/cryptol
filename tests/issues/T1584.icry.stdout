Loading module Cryptol
[1, 2, 3, 4, 5]
